caso/extract/nova.py

```diff
@@ -115,7 +116,13 @@ def _get_keystone_user(self, uuid):
         try:
             user = self.keystone.users.get(user=uuid)
             return user.name
-        except Exception:
+        except keystoneauth1.exceptions.http.Forbidden as e:
```

Comment:
Even if caso is not able to get the user, the accounting records are valid, no?
Is it possible to avoid calling keystone for every record if we get a forbidden here?
And if we really need to have the user I would directly fail here, otherwise what's the point of generating bad records?